released/packages/coq-haskell/coq-haskell.1.0.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "johnw@newartisans.com"
+version: "1.0.0"
```

Comment:
```suggestion
```
The version is already defined by the path name (`coq-haskell.1.1.0`).

Reply:
Where is this? I don't see it in the Coq Package Index.

Reply:
The Coq Package Index website only shows certain aspects of opam metadata. It's maybe easiest to illustrate with an example. Here is the deployed package definition for `coq-aac-tactics` version `8.15.1`. Notice that there is no `version` clause: https://coq.inria.fr/opam/released/packages/coq-aac-tactics/coq-aac-tactics.8.15.1/opam

However, the package *path* clearly contains the version number (`8.15.1`). Hence, including `version` here in the package definition itself is redundant.

Reply:
Ah, I see what you mean.